contact-us.html

```diff
@@ -147,35 +138,90 @@ <h1 class="secondary-heading">CONTACT US</h1>
 
       <!-- Contact info cards  -->
       <ul class="contact-info container" id="contact-info">
-        <li>
-          <img src="./img/btn-icons/contact-location.png" alt="" />
-          <h2>Location</h2>
-          <a href="https://maps.app.goo.gl/qJvwvcxLTHW6qtAg6" target="_blank"
+        <li class="contact-info-option">
+          <svg class="icon contact-info-icon" width="56" height="56">
+            <use xlink:href="sprite.svg#location"></use>
+          </svg>
+          <h2 class="contact-info-title">Location</h2>
+          <a
+            class="contact-details"
+            href="https://maps.app.goo.gl/qJvwvcxLTHW6qtAg6"
+            target="_blank"
             >2163 Phillips Gap Rd West Jefferson</a
           >
         </li>
-        <li>
-          <img src="./img/btn-icons/contact-email.png" alt="" />
-          <h2>Email</h2>
-          <a href="mailto:Info.avitex@gmail.com">Info.avitex@gmail.com</a>
+        <li class="contact-info-option">
+          <svg class="icon contact-info-icon" width="56" height="56">
+            <use xlink:href="sprite.svg#email"></use>
+          </svg>
+          <h2 class="contact-info-title">Email</h2>
+          <a class="contact-details" href="mailto:Info.avitex@gmail.com"
+            >Info.avitex@gmail.com</a
+          >
         </li>
-        <li>
-          <img src="./img/btn-icons/contact-phone.png" alt="" />
-          <h2>Phone</h2>
-          <a href="tel:+16668888">+1 666 8888</a>
+        <li class="contact-info-option">
+          <svg class="icon contact-info-icon" width="56" height="56">
+            <use xlink:href="sprite.svg#phone"></use>
+          </svg>
+          <h2 class="contact-info-title">Phone</h2>
+          <a class="contact-details" href="tel:+16668888">+1 666 8888</a>
         </li>
       </ul>
 
       <!-- Get in touch section -->
       <section class="get-intouch-block container">
         <h2 class="secondary-heading">Get in touch</h2>
-        <p>Browse through the most freuently asked questions</p>
-        <form action="post">
-          <input type="text" placeholder="Name" />
-          <input type="email" placeholder="Enter your email" />
-          <input type="tel" placeholder="Phone Number" />
-          <textarea name="comment" rows="5" placeholder="Message"></textarea>
-          <button type="submit" class="btn-border">SEND MESSAGE</button>
+        <p class="get-intouch-text">
+          Browse through the most freuently asked questions
+        </p>
+        <form action="" method="post" class="get-intouch-form">
+          <fieldset class="form-block">
+            <label class="visually-hidden" for="name">Your full name:</label>
```

Comment:
I'm not sure if this is needed here

Reply:
#fixed

Reply:
I told about hidden label :)

Reply:
I don't understand why no need label??? could you clarify please?

Reply:
It is invisible and does not affect the user’s interaction with the input, so I think it can be removed from the markup

Reply:
Ok, let's leave it